custom_components/waste_collection_schedule/waste_collection_schedule/source/juneavfall_se.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+from datetime import datetime
+import json
+from urllib.parse import urlencode
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "June Avfall & Miljö"
```

Comment:
TITLE differs from text in README.md. Did you use update_docu_links.py to create the changes in README.md and info.md???